package.json

```diff
@@ -11,23 +11,30 @@
     "editor"
   ],
   "scripts": {
+    "test": "mocha-webpack test/",
```

Comment:
Seeing that `mocha-webpack` has a `--webpack-config` option and our Webpack config already includes some environment-specific handling, do you have any opinion on whether it might make sense to merge `webpack.config.test.js` into the base config and change this to:

```
"test": "NODE_ENV=test mocha-webpack --webpack-config webpack.config.js test/"
```

Trying to think of ways to avoid root directory becoming too sprawling with files.